excel/tasks.py

```diff
@@ -1,14 +1,21 @@
-import sys
+from importlib.machinery import SourceFileLoader
 from pathlib import Path
 
-# Add the devutils even if the poetry env isn't setup (to do a 'inv devinstall').
+# Since the `robocorp-devutils` dependency is available in the local Poetry isolated
+#  environment only, we won't have it available with Invoke commands running outside
+#  such environment.
```

Comment:
```suggestion
# environment only, we won't have it available with Invoke commands running outside
# such environment.
```

Reply:
There's a rule in PyCharm where such indentation separates a contiguous comment block from the other comments.

Highlight can be observed with TODO and FIXME:
<img width="655" alt="Screenshot 2024-02-08 at 4 05 50 PM" src="https://github.com/robocorp/robocorp/assets/709053/fcf79ca4-d5f5-4413-b4e2-4c3e12ae24e8">
<img width="678" alt="Screenshot 2024-02-08 at 4 05 57 PM" src="https://github.com/robocorp/robocorp/assets/709053/11e4b2ac-7077-4e44-b8c0-07c18627a5f5">

Reply:
interesting, had no idea this is a normal format